# Serial setup
serial_output_service: /urGovernor/serial_output_service
serial_input_service: /urGovernor/serial_input_service
serial_port: /dev/ttyTHS1
serial_baud_rate: 115200
serial_timeout_ms: 250

# **MAX** Query rate of the controller to make service requests for new weeds [Hz]
controller_overall_rate: 0.125 

# Time for end-effector to perform its duties (in seconds)
end_effector_time_s: 2
